### PR TITLE
readme Ethereum update issue #1003

### PR DESCRIPTION
**Pull request by evanpjensen**
https://github.com/trailofbits/manticore/issues/1003

---
This change is [Reviewable](https://reviewable.io/reviews/trailofbits/manticore/1034)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,100 @@ Manticore can analyze the following types of programs:
...
+    }
+}
+"""
+m = ManticoreEVM()
+
+user_account=m.create_account(balance=1000)
+contract_account=m.solidity_create_contract(contract_src,
+                                            owner=user_account,
+                                            balance=0)
+bar=m.make_symbolic_value(name="bar")
```
Comment (feliam) [inline README.md]: You do not actually need to provide a `name=` there (in case you find it more clear not to)

**Comment by evanpjensen** [comment]
It’s become a habit. Makes it easier to track down that variable in the
smt. I can remove it if you think it improves clarity.

**Comment by feliam** [comment]
It is not used again in the script. I'm not sure if it improves the learning curve for the new guy reading the readme. :man_shrugging:

**Comment by mossberg** [comment]
imo, this buries the "quick start" a tad far down the page. i also think the "further documentation..." and "manticore is beta software..." lines belong at/closer to the bottom

**Comment by evanpjensen** [comment]
I thought about where to put the further documentation. Because it was mostly code related I thought placing it near the code was the best option. The wiki has additional information and I agree should be broken out. To solve the problem of burring other sections under code what do you think about a table of contents with anchor links at the top or moving the code samples to the examples folder?

**Pull request merged**